Snake/Snake/Style.cs

```diff
@@ -9,34 +9,23 @@ namespace Snake
     
     class Style
     {
-        public const String ANSI_RESET = "\u001B[0m";
-        public const String ANSI_BLACK = "\u001B[30m";
-        public const String ANSI_RED = "\u001B[31m";
-        public const String ANSI_GREEN = "\u001B[32m";
-        public const String ANSI_YELLOW = "\u001B[33m";
-        public const String ANSI_BLUE = "\u001B[34m";
-        public const String ANSI_PURPLE = "\u001B[35m";
-        public const String ANSI_CYAN = "\u001B[36m";
-        public const String ANSI_WHITE = "\u001B[37m";
         public void StyleMaze(int[,] maze)
         {
             System.Console.Clear();
 
-            string row = "";
+            string designedMaze = "";
             int rowLength = maze.GetLength(0);
             int colLength = maze.GetLength(1);
```

Comment:
use whole words in variable names unless you really have to shorten a very long one. so colLength should be columnLength